docs/general/descriptions.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Descriptions
+---
+
+# Descriptions
+
+Documentation is first‐class feature of GraphQL, and human-readable
+descriptions can be added into all GraphQL types, fields and
+other definitions.
+
+These are often shown by interactive GraphQL clients (Like
+[GraphiQL](https://github.com/graphql/graphiql) and documentation generators.
+
+The preferred way of adding description in strawberry is using the
+`description=` argument in strawberry decorators:
+
+```python+schema
+@strawberry.enum(description="Example enum")
+class EnumType(Enum):
+    FOO = "foo"
+    BAR = "bar"
+
+@strawberry.type(description="The main GraphQL type")
+class Query:
+    enum: EnumType = strawberry.field(default=EnumType.BAR, description="A dataclass field")
+
+    @strawberry.field(description="A GraphQL field with a resolver and arguments")
+    def resolver(self, arg1: str, arg2: int) -> int:
+        return 1
+
+schema = strawberry.Schema(query=Query)
+---
+"""Example enum"""
+enum EnumType {
+  FOO
+  BAR
+}
+
+"""The main GraphQL type"""
+type Query {
+  """A dataclass field"""
+  enum: EnumType!
+
+  """A GraphQL field with a resolver and arguments"""
+  resolver(arg1: String!, arg2: Int!): Int!
+}
+```
+
+## Formatting
+
+Descriptions should be formatted using Markdown syntax (as specified by
+[CommonMark](http://commonmark.org/))
+
+## Docstrings
+
+It is also possible to generate descriptions from Python docstrings.
+
+Additionally it is possible to add descriptions to enum values and resolver arguments when
+one of the [supported syntaxes is used](https://pypi.org/project/docstring-parser/)
+
+<Note>
+
+This needs to be explicitly enabled with `StrawberryConfig(descriptions_from_docstrings=True)`
```

Comment:
This and the other doc never got updated after `descriptions_from_docstrings` was removed

Reply:
Good catch!